### ISSUE TITLE
The elm-package example source links are outdated (404) 

### ISSUE THREAD

**Issue by PixelPartner** [w0rm/elm-physics#107]

**Comment by w0rm** [comment]
@PixelPartner sorry I’ve been reworking examples to separate them from tests/prototype. I haven’t released the new version yet, the old examples can be found a few commits back. 

I figured out How to solve this issue in the new releases of elm-physics. I will rather link to the tagged revisions, so the examples can always be found in the git history

**Comment by NiavlysB** [comment]
The examples that were moved can be found here: https://github.com/w0rm/elm-physics/tree/main/sandbox/src

**Comment by w0rm** [comment]
@NiavlysB correct! So my goal here is to split examples from the tests / experiments. The examples are rendered with elm-3d-scene while the sandbox code is using my scrappy renderer on top of raw elm-explorations/webgl 

Hopefully things are going to be cleaner when I update the readme and publish a new version of elm-physics

**Comment by w0rm** [comment]
Just released a new version https://package.elm-lang.org/packages/w0rm/elm-physics/latest/
